Fix bitrise.io build

Builds stopped working at bitrise.io, see https://app.bitrise.io/build/25fdf5a87a890fb2.

```
+------------------------------------------------------------------------------+

| (2) install-missing-android-tools@2.2.0                                      |
+------------------------------------------------------------------------------+
| id: install-missing-android-tools                                            |
| version: 2.2.0                                                               |
| collection: https://github.com/bitrise-io/bitrise-steplib.git                |
| toolkit: go                                                                  |
| time: 2019-01-07T17:55:57Z                                                   |
+------------------------------------------------------------------------------+
|                                                                              |
INFO[17:55:57] Start installing (golang) with apt-get       
INFO[17:55:57]  * [OK] Step dependency (go) installed, available. 
Configs:
- GradlewPath: ./gradlew
- AndroidHome: /opt/android-sdk-linux
Preparation
Set executable permission for gradlew
Initialize Android SDK
Ensure android licences
Ensure required Android SDK components
Retrying...
Failed to ensure android components, error: output: To honour the JVM settings for this build a new JVM will be forked. Please consider using the daemon: https://docs.gradle.org/2.14.1/userguide/gradle_daemon.html.
Fabric API Secret found
VersionCode is set to 10000
VersionName is set to 1.0
Fabric API key found
Incremental java compilation is an incubating feature.
FAILURE: Build failed with an exception.
* What went wrong:
A problem occurred configuring project ':app'.
> No toolchains found in the NDK toolchains folder for ABI with prefix: mipsel-linux-android
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
BUILD FAILED
Total time: 17.529 secs
error: exit status 1
|                                                                              |
+---+---------------------------------------------------------------+----------+
| x | install-missing-android-tools@2.2.0 (exit code: 1)            | 94 sec   |
+---+---------------------------------------------------------------+----------+
| Update available: 2.2.0 -> 2.3.4                                             |
| Issue tracker: ...bitrise-steplib/steps-install-missing-android-tools/issues |
| Source: ...://github.com/bitrise-steplib/steps-install-missing-android-tools |
+---+---------------------------------------------------------------+----------+
```

--- a/.bitrise.yml
+++ b/.bitrise.yml
@@ -13,13 +13,27 @@ workflows:
         run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
     - git-clone@4.0.12: {}
     - install-missing-android-tools@2.2.0: {}
-    - gradle-runner@1.8.3:
+    - script@1.1.5:
+        inputs:Switched to shell script from Gradle runner at bitrise (#36)
+        - content: |-
+            #!/usr/bin/env bash
+            # fail if any commands fails
+            set -e
+            # debug log
+            set -x
+
+            unset ANDROID_NDK_HOME
+
+            ./gradlew assembleDebug testDebug jacocoTestReport checkstyle pmd jdepend lintDebug buildDashboard assembleDebugAndroidTest crashlyticsUploadDistributionDebug -PversionCode=$BITRISE_BUILD_NUMBER -PfabricApiKey=$FABRIC_API_KEY -PfabricApiSecret=$FABRIC_API_SECRET
         title: Gradle Build/Test/Analyse/Distribute
-        inputs:
-        - gradle_task: assembleDebug testDebug jacocoTestReport checkstyle pmd jdepend
-            lintDebug buildDashboard crashlyticsUploadDistributionDebug -PversionCode=$BITRISE_BUILD_NUMBER
-            -PfabricApiKey=$FABRIC_API_KEY -PfabricApiSecret=$FABRIC_API_SECRET
-        - mapping_file_exclude_filter: ''
+# disabled because of https://github.com/vgaidarji/ci-matters/issues/35
+#    - gradle-runner@1.8.3:
+#        title: Gradle Build/Test/Analyse/Distribute
+#        inputs:
+#        - gradle_task: assembleDebug testDebug jacocoTestReport checkstyle pmd jdepend
+#            lintDebug buildDashboard crashlyticsUploadDistributionDebug -PversionCode=$BITRISE_BUILD_NUMBER
+#            -PfabricApiKey=$FABRIC_API_KEY -PfabricApiSecret=$FABRIC_API_SECRET
+#        - mapping_file_exclude_filter: ''
     - gradle-coveralls@1.0.1:
         inputs:
         - coveralls_task: coveralls -PversionCode=$BITRISE_BUILD_NUMBER -PfabricApiKey=$FABRIC_API_KEY

